src/parameters.ts

```diff
@@ -1,82 +1,82 @@
-import * as CryptoJS from "crypto-js";
-
 import { hashBitCount } from "./utils";
+import * as crypto from "crypto";
+
+export interface PrimeGroup {
```

Comment:
I thought this was important to add because the Primes and generators are related. Coincidentally, g happened to be 2 for all the primes listed, but if we add more in the future the generator may not always be 2.

Reply:
That's nice.